CommonLibSF/include/SFSE/Impl/PCH.h

```diff
@@ -759,5 +759,5 @@ namespace REL
 #include "RE/IDs_VTABLE.h"
 
 #include "RE/F/FormTypes.h"
-
+#include "RE/M/msvc.h"
```

Comment:
```suggestion

#ifndef __clang__
#include "RE/M/msvc.h"
#endif

```

Reply:
to be clear, this isn't really an "MSVC-only" thing, that's just a name, this is just a reimplementation of these std types that makes sure it works with the RE'd classes that use them.

Reply:
Sure, I just suggested the ifdef to get rid of the build error for clang-cl